assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -107,6 +107,94 @@ var ajaxCallbacks = {
       }
     }
   },
+  apiCollaboratorResponse: {
+    callbacks: {
+      success: function(response, $element, data, helpers, targets, container, type) {
+        var email = $element.find('[name=email]').val();
+        var list = $('[data-collaborator-list]');
+        var row = list.children().first().clone(true);
```

Comment:
can this reference a selector rather than rely on the structure of markup for a specific element?